tests/test_wave_07_NEW.py

```diff
@@ -0,0 +1,77 @@
+import pytest
```

Comment:
OMG And you wrote tests?!?!~~

Reply:
These tests look great!

Reply:
thank you SO much for this feedback, Jared! I'm sorry for taking so long to answer, I'll implement the changes as you suggested. Thanks again and have a great weekend!

Reply:
Ivana, I love that you took the time to respond to this feedback! You are definitely welcome to update your code with the suggestions I offered but I want to make sure you know you aren't expected or required to. 
The feedback is first and foremost feedback for you to incorporate in future code submissions. :)